projects/wireguard/tools/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:edge as tools
```

Comment:
can you use `linuxkit/alpine` - dont see why we should not add to base now, as it doesnt affect anything.